app/views/charge.njk

```diff
@@ -63,60 +63,63 @@
                   <span id="apple-pay-payment-method-divider-word" class="pay-divider--word">or</span></span>
-              </form>
+                </form>
+              </div>
             {% endif %}
             {% if allowGooglePay %}
-              <h2 class="govuk-heading-m">{{ __p("cardDetails.webPayments.payWithGooglePay") }}</h2>
-              
-              {% set GooglePayButtonHTML %}
-                <span class="govuk-visually-hidden">Google Pay</span>
-              {% endset %}
-              
-              <form id="google-pay-container" class="govuk-!-width-three-quarters web-payments-container">
-                <input type="hidden" name="payment-method" value="Google Pay"/>
-                {{
-              govukButton({
-                html: GooglePayButtonHTML,
-                classes: "web-payment-button web-payment-button--google-pay",
-                attributes: {
-                  id: "google-pay-payment-method-submit"
-                }
-              })
-            }}
-                <span id="google-pay-payment-method-divider" class="pay-divider">
+              <div class="google-pay-container">
+                <h2 class="govuk-heading-m">{{ __p("cardDetails.webPayments.payWithGooglePay") }}</h2>
+
+                {% set GooglePayButtonHTML %}
+                  <span class="govuk-visually-hidden">Google Pay</span>
+                {% endset %}
+
+                <form class="govuk-!-width-three-quarters web-payments-container">
+                  <input type="hidden" name="payment-method" value="Google Pay"/>
+                  {{
+                    govukButton({
+                      html: GooglePayButtonHTML,
+                      classes: "web-payment-button web-payment-button--google-pay",
+                      attributes: {
+                        id: "google-pay-payment-method-submit"
+                      }
+                    })
+                  }}
+                  <span id="google-pay-payment-method-divider" class="pay-divider">
                   <span id="google-pay-payment-method-divider-word" class="pay-divider--word">or</span></span>
-              </form>
+                </form>
+              </div>
             {% endif %}
             <div id="spinner" class="hidden"><img src="/images/spinner.gif" alt="{{ __p("authorisation.spinnerAltText") }}"/></div>
 
             <div id="card-details-wrap">
               {% if allowApplePay or allowGooglePay %}
-                <h2 class="govuk-heading-m">{{ __p("cardDetails.enterCardDetails") }}</h2>
-              {% else %}
-                <h1 class="govuk-heading-l">{{ __p("cardDetails.enterCardDetails") }}</h1>
+                <h2 class="govuk-heading-m web-payment-button-heading">{{ __p("cardDetails.enterCardDetails") }}</h2>
               {% endif %}
+              <h1 class="govuk-heading-l non-web-payment-button-heading">{{ __p("cardDetails.enterCardDetails") }}</h1>
```

Comment:
This version of the heading only shows when there are Apple/Google Pay buttons.
So add the `non-web-payment-button-heading` class to it.
This is shown by default.